app/code/Magento/Theme/Console/Command/ThemeUninstallCommand.php

```diff
@@ -375,4 +378,40 @@ private function cleanup(InputInterface $input, OutputInterface $output)
             );
         }
     }
+
+    /**
+     * Enable maintenance mode
+     *
+     * @param OutputInterface $output
+     * @return void
+     */
+    private function enableMaintenanceMode(OutputInterface $output)
```

Comment:
Code duplication can be avoided by extracting this behavior to a separate object

Reply:
How would you do this? Create something like a `MaintenanceModeManagement`? Or would you add this feature to `Magento\Framework\App\MaintenanceMode`? I considered that last one but I don't think `MaintenanceMode` should be responsible for knowing or it should disable maintenance mode.
Do you already have something in mind?